test/framework/src/main/java/org/elasticsearch/test/rest/yaml/ClientYamlTestClient.java

```diff
@@ -108,7 +101,8 @@ public ClientYamlTestResponse callApi(String apiName, Map<String, String> params
             if (restApi.getPathParts().contains(entry.getKey())) {
                 pathParts.put(entry.getKey(), entry.getValue());
             } else {
-                if (restApi.getParams().contains(entry.getKey()) || ALWAYS_ACCEPTED_QUERY_STRING_PARAMS.contains(entry.getKey())) {
+                if (restApi.getParams().contains(entry.getKey()) || restSpec.isGlobalParameter(entry.getKey())
+                        || "ignore".equals(entry.getKey())) {
```

Comment:
not really, it was already part of the previous constant... :) I will add a comment.